docs/flows.md

```diff
@@ -10,11 +10,16 @@ Firebase Genkit provides CLI and Developer UI tooling for working with flows
 ```javascript
 import { defineFlow } from '@genkit-ai/flow';
 
-export const myFlow = defineFlow({ name: 'myFlow' }, async (input) => {
-  const output = doSomethingCool(input);
+export const menuSuggestionFlow = defineFlow(
+  {
+    name: 'menuSuggestionFlow',
+  },
+  async (restaurantTheme) => {
+    const suggestion = makeMenuItemSuggestion(restaurantTheme);
 
-  return output;
-});
+    return suggestion;
+  }
+);
```

Comment:
I'm not sure this example _needs_ to change from the original

Reply:
Do you think I should revert it back? I figured we would just use the menu suggestion theme throughout.

Reply:
My hesitancy comes from this `myFlow` example not necessarily being it's own theme; on it's own it's pretty generic. @kevinthecheung do you have an opinion here?

Reply:
on further thought; @kevinthecheung already approved this change; let's ship it